src/sentry/auth/staff.py

```diff
@@ -250,32 +234,35 @@ def _populate(self) -> None:
                         },
                     )
 
-    def _set_logged_in(self, expires, token, user, current_datetime=None):
+    def _set_logged_in(self, expires: datetime, token: str, user, current_datetime=None):
         # we bind uid here, as if you change users in the same request
         # we wouldn't want to still support staff auth (given
         # the staff check happens right here)
         assert user.is_staff
         if current_datetime is None:
             current_datetime = django_timezone.now()
-        self.token = token
+        self.token: str | None = token
         self.uid = str(user.id)
-        # the absolute maximum age of this session
-        self.expires = expires
         # do we have a valid staff session?
         self.is_valid = True
         # is the session active? (it could be valid, but inactive)
         self._is_active, self._inactive_reason = self.is_privileged_request()
-        self.request.session[SESSION_KEY] = {
-            "exp": self.expires.strftime("%s"),
-            "idl": (current_datetime + IDLE_MAX_AGE).strftime("%s"),
+
+        session_info = {
+            "exp": expires.strftime("%s"),
```

Comment:
are we using this anywhere else in this raw form?

Reply:
I don't think so